.ci/Jenkinsfile

```diff
@@ -271,6 +272,10 @@ void runPytest(String pDockerImage, String markers, String extraDeps, Boolean is
                 name: 'P_JENKINS_USERNAME_PASSWORD_CREDENTIALS',
                 value: "IGNORED_WANDB_USERNAME,WANDB_API_KEY=$wandbCredentialsId",
             ),
+            string(
+                name: 'P_JENKINS_USERNAME_PASSWORD_CREDENTIALS',
+                value: "WANDB_API_KEY=$cometmlCredentialsId",
```

Comment:
should this be different, given the above for wandb?

Reply:
good catch! fixed